src/lib/indexer/__tests__/eventProcessor.test.ts

```diff
@@ -129,7 +165,17 @@ describe('eventProcessor', function () {
               embodied_co2: '42',
             },
           ],
-        ]),
+        ])
+      )
+      const events = [...(result.certificateEvents || [])]
+      expect(events.length).to.equal(1)
+      const event = events[0]
+      expect(event[1]).to.deep.equal({
+        type: 'insert',
+        id: event[0],
+        certificate_id: 'caa699b7-b0b6-4e0e-ac15-698b7b1f6541',
+        occurred_at: blockTime,
+        event: 'issued',
```

Comment:
Does this mean in our React frontend , in the component that renders the timestamp we will have to change from `updated_at` to this ?

Reply:
at some point yes